packages/cozy-stack-client/src/DocumentCollection.js

```diff
@@ -316,13 +317,24 @@ class DocumentCollection {
   /**
    * Use Couch _changes API
    *
-   * @param  {String} since     Starting sequence for changes
-   * @param  {Object} options   { includeDesign: false, includeDeleted: false }
+   * @param  {Object} couchOptions Couch options for changes https://kutt.it/5r7MNQ
+   * @param  {Object} options      { includeDesign: false, includeDeleted: false }
    */
-  async fetchChanges(since, options = {}) {
+  async fetchChanges(couchOptions = {}, options = {}) {
+    let urlParams = ''
+    if (typeof couchOptions !== 'object') {
+      urlParams = `?include_docs=true&since=${couchOptions}`
+      console.warn(
+        'fetchChanges use couchOptions as Object not a string, since is deprecated, please use fetchChanges({include_docs: true, since: 0}).'
+      )
+    } else if (Object.keys(couchOptions).length > 0) {
+      urlParams = `?${map(couchOptions, (value, key) => `${key}=${value}`).join(
```

Comment:
Nit: Some options can be arrays (`doc_ids` for example). Maybe we can use something like [qs](https://www.npmjs.com/package/qs) to be more robust.